src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java

```diff
@@ -243,6 +245,13 @@ private static void extractTestProperties(Project project, Map<String, Object> p
   }
 
   private static void configureJaCoCoCoverageReport(final Test testTask, final boolean addForGroovy, Project project, final Map<String, Object> properties) {
+    project.getTasks().withType(JacocoReport.class, jacocoReportTask -> {
+      SingleFileReport xmlReport = jacocoReportTask.getReports().getXml();
+      if (xmlReport.isEnabled() && xmlReport.getDestination().exists()) {
```

Comment:
Maybe we could log a message when xml report is not enabled?

Reply:
log added